src/pages/Phase_04.svelte

```diff
@@ -74,78 +116,33 @@ Data stored/modified:
         rating: r5,
       },
       {
-        questionText: "How satisfied are you with the decision?",
-        type: "satisfied",
-        rating: r6,
-      },
-    ];
-  } else {
-    questions = [
-      {
-        questionText: "How much pain did you feel?",
-        type: "pain",
-        rating: r1,
-      },
-      {
-        questionText: "How satisfied are you with the decision?",
+        questionText: "How satisfied are you with this outcome?",
         type: "satisfied",
-        rating: r2,
-      },
-      {
-        questionText: `How angry do you feel towards ${$groupStore.D1_name}?`,
-        type: "D1_anger",
-        rating: r3,
-      },
-      {
-        questionText: `How indebted do you feel towards ${$groupStore.D1_name}?`,
-        type: "D1_indebted",
-        rating: r5,
-      },
-      {
-        questionText: `How much gratitude do you feel towards ${$groupStore.D1_name}?`,
-        type: "D1_gratitude",
-        rating: r7,
-      },
-      {
-        questionText: `How much do you want to compensate ${$groupStore.D1_name}?`,
-        type: "D1_compensate",
-        rating: r9,
-      },
-      {
-        questionText: `How angry you feel towards ${$groupStore.D2_name}?`,
-        type: "D2_anger",
-        rating: r4,
-      },
-      {
-        questionText: `How indebted do you feel towards ${$groupStore.D2_name}?`,
-        type: "D2_indebted",
         rating: r6,
       },
-      {
-        questionText: `How much gratitude do you feel towards ${$groupStore.D2_name}?`,
-        type: "D2_gratitude",
-        rating: r8,
-      },
-      {
-        questionText: `How much do you want to compensate ${$groupStore.D2_name}?`,
-        type: "D2_compensate",
-        rating: r10,
-      },
     ];
   }
 
   async function getNextTrial() {
     submitted = true;
-    await saveAPQData(questions);
+    await saveQData(questions);
+    console.log("finished phase-04");
     dispatch("get-next-trial");
   }
 </script>
 
 {#if submitted}
-  <Loading text={"Waiting for other participants..."} />
+  <Loading text={"Waiting for your partner..."} />
 {:else}
   <div class="w-1/2 mx-auto">
     <div class="min-w-full pb-32 text-center">
+      <p class="mb-1 text-4xl">
+        You earned ${earnings} for this trial!
+      </p>
+      <p class="mb-1 text-lg">
+        {outcomeExplanation}
+      </p>
+
       {#each questions as question, i}
```

Comment:
We might want to switch to the standard rating scale for these questions cause I don't think we'll have nearly as many as we did the Max's thesis and we were optimizing a bit for speed there. That'll keep it consistent with other phases of the trial, as I recall some of Max's participants needing a few trials to get used to the MouseRating scale.

Reply:
72cbde6